src/main/java/edu/greenblitz/tobyDetermined/commands/telescopicArm/claw/GripCube.java

```diff
@@ -0,0 +1,28 @@
+package edu.greenblitz.tobyDetermined.commands.telescopicArm.claw;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.wpi.first.wpilibj.Timer;
+
+public class GripCube extends ClawCommand {
+
+    Timer timer;
+    @Override
+    public void initialize() {
+        claw.cubeCatchMode();
+        claw.motorGrip();
+        timer = new Timer();
+        timer.start();
+    }
+
+    @Override
+    public boolean isFinished() {
+        Timer.delay(RobotMap.TelescopicArm.Claw.TIME_OF_GRIP_CONSTANT);
```

Comment:
i believe delay makes the entire robot stop and timer is already initialized here just use hasElapssed